runtime/compiler/control/CompilationThread.cpp

```diff
@@ -12662,8 +12663,9 @@ void J9Method_HT::onClassUnloading(J9ClassLoader *j9classLoader)
       HT_Entry *prev = NULL;
       while (entry)
          {
-         if (NULL == entry->_j9method
-            || J9_CLASS_FROM_METHOD(entry->_j9method)->classLoader == j9classLoader)
+         J9Class *clazz = J9_CLASS_FROM_METHOD(entry->_j9method);
+         if ( J9_ARE_ALL_BITS_SET(clazz->classLoader->gcFlags, 0x2)
```

Comment:
Same concern here - magic numbers are painful when evolving the code

Reply:
Included J9modron.h directly. See https://github.com/AlenBadel/openj9/commit/5095b2f287b4a7a6dcbd248634772daaa299a8f5